### PR TITLE
FOR REVIEW: Summary, readme, stability fee and impact estimations

### PR DESCRIPTION
**Pull request by astronautthis**
Staging PR: https://github.com/makerdao/governance-manual/pull/219
Summary: No staging link
Readme: No staging link
Impact estimations: https://longforwisdom.gitbook.io/new-collection/governance/voting-in-makerdao/impact-estimations
Stability fee: https://longforwisdom.gitbook.io/new-collection/parameter-index/vault-risk/param-stability-fee
Minor correction in the How to vote video page: https://longforwisdom.gitbook.io/new-collection/governance/voting-in-makerdao/how-to-vote

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: >-
+Description: >-
```
Comment (LongForWisdom) [inline README.md:2]: Check that this doesn't break anything on staging. I don't think this is a visible piece of text, it's a key to tell gitbooks to display something in a certain way.

File: governance/how-to-vote.md [hunk 2]
```diff
@@ -4,5 +4,5 @@ The following video is a guide to voting in the Maker Protocol:
 
 {% embed url="https://vimeo.com/649207489" %}
 
->Page last reviewed: -  2022-09-29
->Next review due: -  2023-09-29
\ No newline at end of file
+>Page last reviewed: 2022-09-29
```
Comment (LongForWisdom) [inline governance/how-to-vote.md:7]: imo this is just a video link, if it worked when you checked it, also update the reviewed date.

File: governance/impact_estimations.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Impact Estimations
```
Comment (LongForWisdom) [inline governance/impact_estimations.md:1]: filename should match the convention of using hyphens. so impact-estimations.md

File: governance/impact_estimations.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+If there is any doubt about a given estimation, it will generally be rounded up.
+
+
+
+## Usage
+
+GovAlpha will initially be adding impact estimation tags to the [governance tracker spreadsheet](https://docs.google.com/spreadsheets/d/1LWNlv6hr8oXebk8rvXZBPRVDjN-3OrzI0IgLwBVk0vM/edit#gid=0) and will also be including these tags forum posts that represent RFC MIPs and Signal Requests. An example is shown below.
+
+![](https://i.imgur.com/O6Yyz10.png)
```
Comment (LongForWisdom) [inline governance/impact_estimations.md:36]: We should use an image that doesn't show me as a governance facilitator imo. Also, don't use imgur for links. You need to include the image in the repo and link to it using a relative link.

File: governance/impact_estimations.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+* Anything that adds or removes a trivial amount of money from the surplus buffer will likely be marked as `medium` impact.
+
+**Likely Low Impact**
+* Minor wording changes and/or fixes to existing MIPs will likely be marked as `low` impact.
+
+If there is any doubt about a given estimation, it will generally be rounded up.
+
+
+
+## Usage
```
Comment (LongForWisdom) [inline governance/impact_estimations.md:32]: You should update this section, they are currently used on the voting portal, and I think there are a couple of other place we surface them. I think:
* Forum
* Voting Portal
* Hernando's frontend?
* That tracking sheet.

File: governance/impact_estimations.md (hunk 4 above)
Comment (LongForWisdom) [reply governance/impact_estimations.md:36]: Maybe include images for the voting portal and hernando's dashboard as well?

File: governance/impact_estimations.md (hunk 3 above)
Comment (LongForWisdom) [reply governance/impact_estimations.md:1]: Note remember to add this to the main MOM tracking spreadsheet, and to ensure that it's set to mark itself NEEDS REVIEW in a year.

File: governance/impact_estimations.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+GovAlpha will initially be adding impact estimation tags to the [governance tracker spreadsheet](https://docs.google.com/spreadsheets/d/1LWNlv6hr8oXebk8rvXZBPRVDjN-3OrzI0IgLwBVk0vM/edit#gid=0) and will also be including these tags forum posts that represent RFC MIPs and Signal Requests. An example is shown below.
+
+![](https://i.imgur.com/O6Yyz10.png)
+
+
+In the future, these may be integrated these into the [voting portal](https://vote.makerdao.com) as well.
```
Comment (LongForWisdom) [inline governance/impact_estimations.md:39]: Already happened, as commented above.

File: parameter-index/vault-risk/param-stability-fee.md [hunk 7]
```diff
@@ -40,7 +40,7 @@ Likewise, lower Stability Fees may temporarily or permanently decrease the reven
 
 Adjusting a Stability Fee parameter is a manual process that requires an executive vote. Changes to Stability Fee parameters are subject to the GSM Pause Delay.
 
-The Rates Working Group has been empowered by Maker Governance to propose rate changes based on monetary policy, risk, and competitive landscape. The operations, membership, and historic proposals of this working group can be found on the Maker Governance Forum.
+The Rates Working Group has been empowered by Maker Governance to propose rate changes based on monetary policy, risk, and competitive landscape. The operations, membership, and historic proposals of this working group can be found on the [Maker Governance Forum](https://forum.makerdao.com/).
```
Comment (LongForWisdom) [inline parameter-index/vault-risk/param-stability-fee.md:43]: This has been called the Maker Open Market Committee for a good while, we should use that term, imo. Also, I'm like 90% sure there is a tag on the forum with all MOMC related stuff under it. We should link directly to that tag.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
You also need to update the preamble on the Stability Fee Parameter page. There is a checklist of stuff to check for parameter docs on the MOM group work tracking sheet.

**Comment by astronautthis** [comment]
@LongForWisdom not sure where to put impact-estimations.md on the navigation pane. I put it under "voting-in-makerdao" for now but feel free to move it if you think it fits better elsewhere. 

Also, could you take a look at the images in impact-estimations.md if they're okay? Thanks!

**Comment by LongForWisdom** [comment]
You've somehow got some other stuff mixed up in here.

File: SUMMARY.md [hunk 8]
```diff
@@ -28,6 +28,7 @@
   * [Off-Chain Governance](governance/off-chain-governance.md)
   * [Approval Voting Guide](governance/approval-voting-guide.md)
   * [Video Guide To Voting In The Maker Protocol](governance/how-to-vote.md)
+  * [Impact Estimations](governance/impact-estimations.md)
```
Comment (LongForWisdom) [inline SUMMARY.md:31]: I'm not sure where else we'd put this, but also doesn't quite feel like it fits. Hm. Definitely wondering if we need to rearrange the Governance section.

File: governance/impact-estimations.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
+# Impact Estimations
+
+GovAlpha applies impact estimations to active governance items (MIPs and Signal Requests). The goal of these estimations is to:
+
+* Provide delegates, core unit members and the wider community with an idea of how important it is that they pay attention to a given governance item given limited time.
+* Provide appropriate context for GovAlpha members conducting internal reviews of governance items.
```
[the remark this answers is not in the document]
Comment (astronautthis) [reply governance/impact-estimations.md:6]: You mean the second bullet point about internal reviews, yeah? And keep line 3 that says "GovAlpha applies impact estimations" as is.

File: governance/impact-estimations.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+* Items that may lead to later decisions that add or remove risk and/or add or remove a non-trivial amount of DAI from the surplus buffer will likely be marked as `medium` impact.
+* Anything that adds or removes a trivial amount of money from the surplus buffer will likely be marked as `medium` impact.
+
+**Likely Low Impact**
+* Minor wording changes and/or fixes to existing MIPs will likely be marked as `low` impact.
+
+If there is any doubt about a given estimation, it will generally be rounded up.
+
+
+## Usage
```
[the remark this answers is not in the document]
Comment (astronautthis) [reply governance/impact-estimations.md:29]: I'm not sure exactly why this is happening but notice the following two commits and the image of the Governance tracker spreadsheet:

1. https://github.com/astronautthis/governance-manual/blob/8b2c29f58cc49328c9fc7abc2be9d6079e6f6ffd/governance/impact-estimations.md

2. https://github.com/astronautthis/governance-manual/blob/e1b1046e492183ea33dd3afcc788fb2da562a205/governance/impact-estimations.md

The image size does change in the two files but I checked that the staging link size stays the same (the small version). Maybe there's some image sizing override happening on GitBook?

[the remark this answers is not in the document]
Comment (astronautthis) [reply governance/impact-estimations.md:29]: The issue is described here: https://github.com/Open-Systems-Pharmacology/docs/issues/60

File: SUMMARY.md (hunk 8 above)
Comment (astronautthis) [reply SUMMARY.md:31]: Yes, I think we should.

File: governance/impact-estimations.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/impact-estimations.md:29]: So, I think you can add whitespace to make them more square, but agree it's a sucky way to 'fix' it.

[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/impact-estimations.md:29]: Either that or take screenshots that are more square

File: parameter-index/vault-risk/param-stability-fee.md [hunk 11]
```diff
@@ -40,7 +40,8 @@ Likewise, lower Stability Fees may temporarily or permanently decrease the reven
 
 Adjusting a Stability Fee parameter is a manual process that requires an executive vote. Changes to Stability Fee parameters are subject to the GSM Pause Delay.
```
Comment (LongForWisdom) [inline parameter-index/vault-risk/param-stability-fee.md:41]: You should link to the GSM page from here.

File: governance/impact-estimations.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (astronautthis) [reply governance/impact-estimations.md:29]: Fixed now

**Comment by astronautthis** [comment]
Adding Emergency Shutdown: [Staging link](https://longforwisdom.gitbook.io/new-collection/module-index/module-emergency-shutdown)

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**